src/deco-button.cpp

```diff
@@ -38,9 +38,14 @@ void button_t::set_hover(bool is_hovered)
     {
         if (is_hovered)
         {
-            this->hover.animate(HOVERED);
+            if (std::string(theme.button_minimize_hover_image).empty() || std::string(theme.button_maximize_hover_image).empty() || std::string(theme.button_close_hover_image).empty()) {
+                this->hover.animate(HOVERED);
+            } else {
+                this->update_texture();
```

Comment:
Am I understanding this correctly that on hover, the hover image is shown without an animation between buttons? If so, we need to fix it so there is an animation between icons on hover/leave.